### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by sw0277** [T-Troll/alienfx-tools#330] [Ideas]
Have you ever entertained the idea of implementing a "Dark Mode"? I know its just cosmetic and I'm sure you have more important things to work on, I just think it would be a nice touch and I'm sure it would be appreciated by many.

**Comment by T-Troll** [comment]
You know, i'll try to keep apps as small and simple as possible. No pictures and other nice, but resource-consuming features, just plain Windows GUI functionality.
I got your idea, but i didn't see high demand for it right now, but significant effort. So it's noted, but postponed for now.

**Comment by sw0277** [comment]
That makes sense, fair enough.
